atomic-structure/content.md

```diff
@@ -0,0 +1,25 @@
+# Radioactive Decay
+#### Learning objective
+Basic understanding of atomic structure
+
+How atomic structure relates to radioactivity
+___
```

Comment:
This horizontal line might not be necessary if the next section were a higher-level heading.